map/change-basemap/src/main/java/com/esri/samples/change_basemap/ChangeBasemapSample.java

```diff
@@ -53,27 +51,32 @@ public void start(Stage stage) {
       stage.setScene(scene);
       stage.show();
 
+      // authentication with an API key or named user is required to access basemaps and other location services
+      String yourAPIKey = System.getProperty("apiKey");
+      ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
+
       // creates a map view
       mapView = new MapView();
 
-      // setup listview of basemaps
-      ListView<Basemap.Type> basemapList = new ListView<>(FXCollections.observableArrayList(Basemap.Type.values()));
-      basemapList.setMaxSize(250, 150);
+      // setup a list view of basemap styles
+      ListView<BasemapStyle> basemapStyleListView = new ListView<>(FXCollections.observableArrayList(BasemapStyle.values()));
+      basemapStyleListView.setMaxSize(250, 150);
 
-      // change the basemap when list option is selected
-      basemapList.getSelectionModel().selectedItemProperty().addListener(o -> {
-        String basemapString = basemapList.getSelectionModel().getSelectedItem().toString();
-        map = new ArcGISMap(Basemap.Type.valueOf(basemapString), LATITUDE, LONGITUDE, LOD);
+      // change the basemap when a list option is selected
+      basemapStyleListView.getSelectionModel().selectedItemProperty().addListener(o -> {
+        BasemapStyle selectedBasemapStyle = basemapStyleListView.getSelectionModel().getSelectedItem();
+        map = new ArcGISMap(selectedBasemapStyle);
         mapView.setMap(map);
+        mapView.setViewpoint(new Viewpoint(57.5000, -5.0000, 10000000.0));
```

Comment:
LOD 6 could be replaced by scale 9244648 (rounded down to 9240000 should be ok).